packages/desktop-client/src/hooks/useFormat.ts

```diff
@@ -155,7 +155,7 @@ export function useFormat(): UseFormatResult {
       const styledAmount =
         position === 'after'
           ? `${valueWithoutSign}${space}${currencySymbol}`
-          : `${currencySymbol}${space}${valueWithoutSign}`;
+          : `\u202A${currencySymbol}\u202C${space}${valueWithoutSign}`;
```

Comment:
These unicode characters force LTR. Without them, Arabic characters (and other RTL characters) get rearranged to the end of the string.

Not technically required for this change, but I found it when I was testing the Arabic symbol instead of the latin equivalent. It'll make it easier if we add any more in the future.